src/ContaoManager/Plugin.php

```diff
@@ -34,9 +34,7 @@ public function getBundles(ParserInterface $parser): array
 
     public function getRouteCollection(LoaderResolverInterface $resolver, KernelInterface $kernel): ?RouteCollection
     {
-        return $resolver
-            ->resolve(__DIR__.'/../Resources/config/routing.yml')
-            ->load(__DIR__.'/../Resources/config/routing.yml')
-        ;
+        return $resolver->resolve('@ContaoYouTubeSyncBundle/src/Action')
+            ->load('@ContaoYouTubeSyncBundle/src/Action');
```

Comment:
```suggestion
        return $resolver
            ->resolve('@ContaoYouTubeSyncBundle/src/Action')
            ->load('@ContaoYouTubeSyncBundle/src/Action')
        ;
```

Does this really work? 🤔 Usually you would have to define whether the routes are registered via attributes or annotations, no?